storageClient/api.go

```diff
@@ -43,13 +44,27 @@ type Group struct {
 	PublicKey    string    `json:"public_key"`
 	CreatedAt    time.Time `json:"created_at"`
 	LastModified time.Time `json:"last_modified"`
+	Index        int64     `json:"idx"`
```

Comment:
This actually shouldn't get returned / exposed via our API, its an internal implementation detail of our db and pagination system.